boards/common/bossac.board.cmake

```diff
@@ -1,2 +1,2 @@
 set_ifndef(BOARD_FLASH_RUNNER bossac)
-board_finalize_runner_args(bossac) # No default arguments to provide.
+board_finalize_runner_args(bossac "--bossac=${ZEPHYR_SDK_INSTALL_DIR}/sysroots/x86_64-pokysdk-linux/usr/bin/bossac")
```

Comment:
The BOARD should be agnostic to the SDK/toolchain used.

So this should be:

--bossac=${BOSSAC}

and then some toolchain script should set BOSSAC appropriately. Perhaps from zephyr/cmake/toolchain/zephyr.cmake